proposals/dao/tip_119.ts

```diff
@@ -0,0 +1,132 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  PcvStats,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { BigNumber } from 'ethers';
+import { overwriteChainlinkAggregator } from '@test/helpers';
+
+const toBN = BigNumber.from;
+
+/*
+
+Tribal Council Proposal: TIP-119: Add gOHM to Collaterisation Oracle
+
+1. Deploy gOHM oracle
+2. Set oracle on collaterisation oracle
+3. Add gOHM to CR 
+
+*/
+
+const fipNumber = 'tip_119';
+let pcvStatsBefore: PcvStats;
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  ////////////// 1. Create and deploy gOHM USD oracle
+  const chainlinkOracleWrapperFactory = await ethers.getContractFactory('ChainlinkOracleWrapper');
```

Comment:
We should refactor the ChainlinkOracleWrapper to take a time parameter to check the staleness of the oracle

Reply:
Should probably be optional. If passed as 0, then don't check, if non-zero then check